service/app.js

```diff
@@ -0,0 +1,332 @@
+import express from 'express'
+import mongoose from 'mongoose'
+import cors from 'cors'
+
+import { Deck } from './models/Deck.js'
+import { User } from './models/User.js'
+
+const app = express()
+const port = 8000
+
+// Connect to MongoDB
+
+const connectionString = `mongodb+srv://krishana:krishana@cluster0.m3dzd.mongodb.net/Cards?retryWrites=true&w=majority`
```

Comment:
You MUST NOT commit passwords and usernames into code.

Reply:
It has been updated, Thank you.